plot/plot_lmbench

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
```

Comment:
thanks, when will this tool be used? and what is the difference between this tool and the plot function in test script?

Reply:
the function of this script is to plat the graph of 2 set of data for comparison. 
For example, if i have 2 different run of lm_mem_rd on different machine, i will have 2 set of lmbench data (raw data), then i can use this tool to plot the graph for visualization.
The plot function on the test script only plot the graph with 1 set of data.

Reply:
got it, is it possible to submit this as a 2nd patch, and add this usage as change log? similar to what you write here

plot/plot_lmbench: to plot a graph by combining 2 set of data.
Usage: plot_lmbench <data file 1> <data file 2>

As it can be considered out of the test case update, but a tool for data analysis.

Reply:
Noted. plot_lmbench is removed and submmit as another PR: https://github.com/intel/lkp-tests/pull/162